pkg/reconciler/v1alpha1/clusteringress/clusteringress_test.go

```diff
@@ -359,6 +359,9 @@ func TestGlobalResyncOnUpdateGatewayConfigMap(t *testing.T) {
 		t.Fatalf("failed to start cluster ingress manager: %v", err)
 	}
 
+	servingInformer.WaitForCacheSync(stopCh)
+	sharedInformer.WaitForCacheSync(stopCh)
```

Comment:
Should we wait on the informer directly rather then on the factory? I think it would make possible errors on cache sync more visible.
I mean the approach we use in most _main_ files, something like: 
```
	informerSyncs := []cache.InformerSynced{
		clusterIngressInformer.Informer().HasSynced,
		virtualServiceInformer.Informer().HasSynced,
	}
	for i, synced := range informerSyncs {
		if ok := cache.WaitForCacheSync(stopCh, synced); !ok {
			logger.Fatalf("failed to wait for cache at index %d to sync", i)
		}
	}
```

Reply:
actually thinking about it - its a test with fake clients, no error should happen there on the wait... so we probably don't need to handle them

Reply:
Right, this is solely to ensure things have actually gotten up before we start throwing entities at the informers.